src/components/ActionButton/styles.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import {Button as CustomButton} from '../CustomButton/styles';
+
+export const IconWrapper = styled(CustomButton)`
```

Comment:
Is there an easier/prettier way to do this? I mean merge styles w/ a CustomButton. Should this size be fixed?